app/src/main/java/openfoodfacts/github/scrachx/openfood/features/ImagesManageActivity.java

```diff
@@ -87,7 +87,7 @@
     private static final int REQUEST_UNSELECT_IMAGE_AFTER_LOGIN = 4;
     public static final int REQUEST_EDIT_IMAGE = 1000;
     private static final int REQUEST_CHOOSE_IMAGE = 1001;
-    private static final List<ProductImageField> TYPE_IMAGE = Arrays.asList(ProductImageField.FRONT, ProductImageField.INGREDIENTS, ProductImageField.NUTRITION);
+    private static final List<ProductImageField> TYPE_IMAGE = Arrays.asList(ProductImageField.FRONT, ProductImageField.INGREDIENTS, ProductImageField.NUTRITION, ProductImageField.PACKAGING);
```

Comment:
I would rename the field to IMAGE_TYPES and make it an array. I don't think we need list operations. 

Also, if possible, I would move the field to the ApiFields class. 

What do you think?

Reply:
Resolved by #59f2681